src/theme/material/variables.css.d.ts

```diff
@@ -0,0 +1,2 @@
+declare const styles: {};
```

Comment:
can you please remove the `css.d.ts` files from your pr

Reply:
Seems like we need them. The build fails without them.

Reply:
Only `css.d.ts` files from new stylesheets remain now.